packages/salesforcedx-utils/test/jest/cli/checkCliVersion.test.ts

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
I don't see tests for the getSfdxCliVersion or get SfCliVersion functions.

Reply:
I originally tried to write tests for getSfdxCliVersion() and getSfCliVersion() functions but found that they're dependent on the version installed on the computer and thus I can't put definitive values to check for.  For example, suppose I have SF v2.14.1 installed on my computer.  In order to get the tests to pass, both getSfdxCliVersion() and getSfCliVersion() need to check for SF v2.14.1.  But the CLI team made a release and the next person to run the tests has SF v2.15.0.  Because v2.14.1 ≠ v2.15.0, the second person has two failing tests, even though they have a valid CLI version.  That's just one example - a third person can have SFDX v7 installed, which is another case.  It's difficult to test getSfdxCliVersion() and getSfCliVersion() functions unless we just test that _something_ is returned, regardless of the value.

Reply:
@daphne-sfdc cant you use mocks?

Reply:
@diyer I didn't think of that.  Let me check the other unit test classes to see how they use mocks and try to implement mocking in this test class.